pkg/transport/generic_transport_controller.go

```diff
@@ -615,3 +802,9 @@ func cleanObject(object *unstructured.Unstructured) *unstructured.Unstructured {
 
 	return objectCopy
 }
+
+func MapCopyInto[Key comparable, Val any](dest, src map[Key]Val) {
+	for key, val := range src {
+		dest[key] = val
+	}
+}
```

Comment:
if we switch to using configmap, this is not needed.
data field of configmap is a map[string]string.

Reply:
Right, we need this func because the current definition involves merging two maps.